pages/t/[link].tsx

```diff
@@ -0,0 +1,50 @@
+
+//Get the links and filter to the corresponding link
+// Redirect to the actual link
+
+import { GetServerSideProps } from 'next';
+import { useRouter } from 'next/router';
+import React from 'react';
+import Layout from '../../components/Layout';
+import styles from '../../styles/Page.module.scss';
+import {linksQuery} from '../../utils';
+
+export default function LinkPage(): JSX.Element {
+  const router = useRouter();
+
+  if (typeof window !== 'undefined' && router.isFallback) {
+    void router.push('/');
```

Comment:
same with before.. not sure if this is correct

Reply:
lets deal with this in another PR

Please make an issue to address the isFallback routing concern